.azure-pipelines/code-quality.yml

```diff
@@ -0,0 +1,46 @@
+trigger:
+- master
+
+pool:
+  vmImage: 'ubuntu-16.04'
+
+container: ninhu/r-sdk-build:latest
+
+steps:
+  - task: Bash@3
+    inputs:
+      targetType: 'inline'
+      script: |
+        Rscript './.azure-pipelines/scripts/validate_copyright_header.R' 'R'
+    displayName: 'Validate copyright header'
+
+  - task: Bash@3
+    inputs:
+      targetType: 'inline'
+      script: |
+        pip install --user azureml-sdk
+        R -e 'dir.create(Sys.getenv("R_LIBS_USER"), recursive = TRUE);
+              .libPaths(Sys.getenv("R_LIBS_USER"));
+              # build and install R sdk
+              devtools::install_deps(upgrade = FALSE);
+              package_location <- devtools::build();
+              install.packages(package_location, repos = NULL)'
+    displayName: 'Build and Install SDK'
```

Comment:
I think we can remove this step to save some time as R CMD check would make sure SDK is build-able.